src/main/perf/LineFileDocs.java

```diff
@@ -553,13 +581,19 @@ public Document nextDoc(DocState doc) throws IOException {
       if (spot2 == -1) {
         throw new RuntimeException("line: [" + line + "] is in an invalid format !");
       }
-      spot3 = line.indexOf(SEP, 1 + spot2);
+      int spot3 = line.indexOf(SEP, 1 + spot2);
       if (spot3 == -1) {
-        spot3 = line.length();
+        throw new RuntimeException("line: [" + line + "] is in an invalid format !");
```

Comment:
Hmm, is this the exceptions users with the "old" line docs file cached will hit?  If so, could we improve the message to suggest downloading the newer version from XYZ location?  Let's make sure to copy up the new line docs file to home.apache.org/~mikemccand before we push this.  I'd also like to confirm I can generate the binary file and nightly benchmarks will be happy.  I'll try to test that path soon.

Reply:
Added new exception message.